pravega-client-examples/src/main/java/io/pravega/example/secure/SecureReader.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.secure;
+
+import io.pravega.client.ClientConfig;
+import io.pravega.client.ClientFactory;
+import io.pravega.client.admin.ReaderGroupManager;
+import io.pravega.client.stream.*;
+import io.pravega.client.stream.impl.DefaultCredentials;
+import io.pravega.client.stream.impl.JavaSerializer;
+
+/**
+ * This class demonstrates how to configure a Pravega reader application for:
```

Comment:
The `pravega-standalone` (or any other Pravega deployment) requires to be configured with security options, right? If so, please mention that as well as how to use the sample with `pravega-standalone` with security enabled.

Reply:
That sounds good. Or, maybe I should reference the documentation that shows how to enable security for standalone since the configuration and information are subject to change. Will do.

Reply:
Great, thanks.

Reply:
Right now, I mentioned the following, but couldn't reference a specific document. There is a [PR 3428](https://github.com/pravega/pravega/pull/3428) in `pravega/pravega` for Standalone TLS, which is pending merge. Also, there doesn't seem to be documentation for enabling Auth in standalone (will create an issue for that). I could provide a reference for https://github.com/pravega/pravega/tree/master/documentation/src/docs/security, but that probably won't help either. 

> First, ensure that the Pravega cluster that the applications are going to work with have both TLS and Auth (authentication and authorization) enabled. See Pravega documentation for how to enable TLS and Auth.